Week1/homework/app.js

```diff
@@ -1,11 +1,84 @@
-'use strict';
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+'use strict'; {
```

Comment:
I would place the opening curly brace on a new line. It has no relation `'use strict';`.

Reply:
thank you, I actually thought they are related I will fix it.